roles/kubernetes-apps/heapster/templates/heapster-deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: heapster
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        task: monitoring
+        k8s-app: heapster
+    spec:
+      containers:
+      - name: heapster
+        image: gcr.io/google_containers/heapster-amd64:v1.3.0-beta.1
```

Comment:
image repo and tag should be be configurable vars with defaults in roles/kubernetes-apps/heapster/defaults/main.yml